specs/deposits.md

```diff
@@ -257,7 +257,7 @@ The deposit contract handles two special cases:
 
 If the caller is a contract, the address will be transformed by adding
 `0x1111000000000000000000000000000000001111` to it. This is the reverse of the
-transformation described in the [deposits spec](./withdrawals.md#address-aliasing). This prevents attacks in which a
+transformation described in the [withdrawals spec](./withdrawals.md#address-aliasing). This prevents attacks in which a
```

Comment:
Note that we do not do this transformation for withdrawals anymore. Another spec change to watch out for.

Reply:
Why do we not do it for withdrawals anymore?

Reply:
Can we add a rationale section to the address aliasing (and I guess explain why we only need it one way)?

Reply:
Also, just noting that the issue says

> Links to the “deposits spec” throughout the documentation should read
“withdrawals spec” instead.

But I double checked and only see one instance of this error.